lib/hammer_cli_foreman_ansible/command_extensions/inventory.rb

```diff
@@ -0,0 +1,118 @@
+            field :description, _('Description')
+          end
+
+          collection :foreman_interfaces, _('Network interfaces') do
+            field :name, _('Interface Name')
+            field :identifier, _('Identifier')
+            field :attrs, _('Attributes'), Fields::Field, hide_blank: true
+            field :mac, _('MAC address')
+            field :ip, _('IPv4 address'), Fields::Field, hide_blank: true
+            field :ip6, _('IPv6 address'), Fields::Field, hide_blank: true
+            field :fqdn, _('FQDN')
+            field :virtual, _('Virtual'), Fields::Boolean
+            field :link, _('Link'), Fields::Boolean
+            field :managed, _('Managed'), Fields::Boolean
+            field :primary, _('Primary'), Fields::Boolean
+            field :provision, _('Provision'), Fields::Boolean
+            field :subnet6, _('IPv6 subnet')
+            field :tag, _('Tag')
+            field :attached_to, _('Attached to')
+            field :type, _('Type')
+            field :attached_devices, _('Attached devices')
+            label _('Subnet') do
+              field :name, _('Name')
+              field :network, _('Network')
+              field :mask, _('Mask')
+              field :gateway, _('Gateway')
+              field :dns_primary, _('Primary DNS')
+              field :dns_secondary, _('Secondary DNS')
+              field :from, _('From')
+              field :to, _('To')
+              field :boot_mode, _('Boot mode')
+              field :ipam, _('IPAM')
+              field :vlanid, _('Vlanid')
+              field :mtu, _('MTU')
+              field :nic_delay, _('Nic delay')
+              field :network_type, _('Network type')
+              field :description, _('Description')
+            end
+          end
+
+          collection :foreman_users, _('Foreman Users'), numbered: false do
+            field :firstname, _('First name')
+            field :lastname, _('Last name')
+            field :mail, _('Email')
+            field :description, _('Description')
+            field :fullname, _('Full name')
+            field :ssh_authorized_keys, _('SSH authorized keys')
+          end
+
+          collection :foreman_ansible_roles, _('Foreman Ansible Roles'), numbered: false do
+            field :name, _('Role')
+          end
+        end
+      end
+
+      before_print do |data, cmd_obj|
+        unless cmd_obj.option_as_json?
+          new_data = data['all']['hosts'].each_with_object([]) do |hostname, arr|
+            data['_meta']['hostvars'][hostname]['foreman']['foreman_users'] = data['_meta']['hostvars'][hostname]['foreman']['foreman_users']&.map { |u| u[1] }
+            data['_meta']['hostvars'][hostname]['foreman_ansible_roles'] =  data['_meta']['hostvars'][hostname]['foreman_ansible_roles']&.map { |r| { 'name' => r } }
+            data['_meta']['hostvars'][hostname].map { |label| data['_meta']['hostvars'][hostname]['foreman'][label[0]] = label[1] }
```

Comment:
If we remove `['foreman']` from 

```ruby
# ...
}.merge(data['_meta']['hostvars'][hostname]['foreman'])
# ...
```

Then we don't need to migrate the information under `'foreman'` key. We could simply extract information under that key in fields definition:

```ruby
# ...
from 'foreman' do
  field :name
  # ...
  collection :foreman_users
  # ...
  collection :foreman_ansible_roles
end
```

And thus we can remove this line.